### ISSUE TITLE
Great, thanks! Let's discuss this for the bar chart in a separate issue.

### ISSUE THREAD

**Issue by AlisonVilela** [swimlane/ngx-charts#1046]
_Originally posted by @marjan-georgiev in https://github.com/swimlane/ngx-charts/pull/422#issuecomment-309456263_

**Comment by damasioalexandre** [comment]
This would be an awesome feature!

**Comment by jackled** [comment]
Yes, absolutely.  Any plans to when this is coming?  The research project I'm working on requires background reference lines in bar graphs.  I'm certain that this feature would make a number of people very happy.

**Comment by AlisonVilela** [comment]
I implemented this feature locally a while ago, because I needed it so much, I can do a PR with these modifications

<img width="724" alt="BarChart" src="https://user-images.githubusercontent.com/6244639/54208652-13024600-44bb-11e9-87df-a17b4d57a5d9.png">

**Comment by sDeVette** [comment]
@AlisonVilela I would love to see a PR for this, or just your implementation would also be very helpful! Thanks in advance

**Comment by rube-de** [comment]
any updates on this?

**Comment by TonySpegel** [comment]
@AlisonVilela can you please post your solution?

**Comment by EmreAkkoc** [comment]
@AlisonVilela would you mind sharing your implementation ?

**Comment by kolya-kra** [comment]
I faced the same issue. My solution might not be the cleanest but it worked. You can create a new component and copy the ngx-bar-chart component from the ngx-charts-Module into it.
Then add the following 3 lines to the `ngx-charts-y-axis`:
```
[referenceLines]="referenceLines"
[showRefLines]="showRefLines"  
[showRefLabels]="showRefLabels"
```

**Comment by EvaristoPerez** [comment]
Thanks kolya-kra I just did this with stacked bar chart and works like a charm. If somebody need it I can send more info.

![image](https://user-images.githubusercontent.com/10392125/113200181-9ff8d300-922d-11eb-8855-35d23f0bcd5b.png)

**Comment by nghiado** [comment]
Can we support this?

**Comment by yusukemetroly** [comment]
@EvaristoPerez hi can you please share what you did? We are running into the same problem. thanks!

**Comment by carlosamn** [comment]
@EvaristoPerez Hi, is there any repo where we can check it out ? or at least any guide? Thanks

**Comment by carlosamn** [comment]
@kolya-kra Hi, Can you share any progress or support us with a guide pls? Thanks!

**Comment by kallekj** [comment]
Hi, @EvaristoPerez  and @kolya-kra! Is it possible for you to share a gist with the code? Many thanks!

**Comment by bobbyg603** [comment]
We've published a version of ngx-charts that includes @kedij777's reference line support. You can use it by installing `@bugsplat/ngx-charts@20.7.0`.
